### PR TITLE
Fix some python error after the refactor of the table linkbase

### PR DESCRIPTION
**Pull request by RobinetDenisAcsone**
Pull request for https://github.com/Arelle/Arelle/pull/296

#### Reason for change
When testing the table linkbase refactoring, I encounter some exception, like some uninitialized variables

#### Description of change
This PR correct the easy errors that I have seen 

#### Steps to Test
Run the conformance for table linkbase

**review**:
@hefischer

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: arelle/RenderingResolution.py [hunk 1]
```diff
@@ -180,8 +181,10 @@ def jsonStrctMdlEncoder(obj, indent="\n"):
             # print(str(o))
             return o
         raise TypeError("Type {} is not supported for json output".format(type(obj).__name__))
-    with io.open(r"/Users/hermf/temp/test.json", 'wt') as fh:
-        json.dump(strctMdlTable, fh, ensure_ascii=False, indent=2, default=jsonStrctMdlEncoder)
+
+    if TRACE_TABLE_STRUCTURE:
+        with io.open(r"/Users/hermf/temp/test.json", 'wt') as fh:
+            json.dump(strctMdlTable, fh, ensure_ascii=False, indent=2, default=jsonStrctMdlEncoder)
```
Comment (RobinetDenisAcsone) [inline arelle/RenderingResolution.py:187]: @hefischer I putted this code in an if, to make it work on other computer. If this PR is merge, you will need to change the TRACE_TABLE_STRUCTURE variable to have the same behavior as before

File: arelle/ViewFileRenderedGrid.py [hunk 2]
```diff
@@ -730,7 +730,7 @@ def yAxisByCol(self, leftCol, row, yParentStrctNode, renderNow, atTop):
                 else:
                     row = nextRow
                 if nestRow > nestedBottomRow:
-                    nestedBottomRow = nestRow + (isNonAbstract and not childrenFirst)
+                    nestedBottomRow = nestRow + isNonAbstract # and not childrenFirst)
```
Comment (RobinetDenisAcsone) [inline arelle/ViewFileRenderedGrid.py:733]: Not quite certain of this change, I don't know if this variable is still needed or not, it's not initialized and the code that used it before (below)  is in a comment

**Comment by hermfischer** [comment]
+1

**Review by austinmatherne-wk — APPROVED** [review]

**Pull request merged**